app/src/main/java/org/jellyfin/androidtv/ui/startup/ListServerFragment.kt

```diff
@@ -76,26 +82,30 @@ class ListServerFragment : RowsSupportFragment() {
 	}
 
 	private fun buildRows(servers: Map<Server, Set<User>>) {
-		val rowAdapter = ArrayObjectAdapter(CustomListRowPresenter())
-
 		servers.forEach { (server, users) ->
-			Timber.d("Adding server row %s", server.name)
+			val exists = rowAdapter.any { it.id == server.id.mostSignificantBits and Long.MAX_VALUE }
```

Comment:
Also it maybe worth adding a comment to explain what `server.id.mostSignificantBits and Long.MAX_VALUE` is doing here as it’s not immediately clear.

Reply:
Done